hnn_core/tests/test_network.py

```diff
@@ -70,36 +66,41 @@ def test_network():
             assert set(conn['gid_pairs'].keys()) == set(
                 net.external_drives[src_type]['conn'][target_type]['src_gids'])
 
-    assert len(net.gid_ranges['bursty1']) == 1
+    # Check drive dict structure for each external drive
     for drive in net.external_drives.values():
+        # Check that connectivity sources correspond to gid_ranges
+        this_src_gids = set([gid for drive_conn in
+                             drive['conn'].values() for
+                             gid in drive_conn['src_gids']])  # NB set: globals
```

Comment:
could we use `net.connectivity` instead for this test? I am anticipating that tests which use `drive['conn']` will start failing once we unify the drive and local connectivity, so this might save us some work then

Reply:
It's unclear to me which list items in `net.connectivity` map to any given drive. Unless there is a quick and easy way accomplish this, I'm afraid this will add more confusion to the test rather than simplicity.

Reply:
If this gets merged before #367, I can update the tests there.

Reply:
Sounds good, thanks @ntolley!